azext_iot/tests/iothub/core/test_iot_hub_unit.py

```diff
@@ -82,6 +83,24 @@ def service_client(self, mocked_response, get_mgmt_client):
 
         yield mocked_response
 
+    @pytest.fixture(params=[(400, BadRequestError)])
```

Comment:
can we also test with 403? - aka what happens if you provide an invalid sas key and I assume that would be more common than the 400

Reply:
Done